eng/pipelines/runtime.yml

```diff
@@ -570,7 +571,6 @@ jobs:
       testScope: innerloop
       liveRuntimeBuildConfig: release
       dependsOnTestBuildConfiguration: ${{ variables.debugOnPrReleaseOnRolling }}
-      dependsOnTestArchitecture: x64
```

Comment:
You shouldn't remove this. The tests are not arch specific and are always built on x64.